x-pack/legacy/plugins/ml/public/jobs/jobs_list/jobs.js

```diff
@@ -13,6 +15,14 @@ import { JobsListView } from './components/jobs_list_view';
 export const JobsPage = (props) => (
   <Fragment>
     <NavigationMenu tabId="jobs" />
+    <EuiTitle style={{ padding: '16px 16px 0 16px' }}>
```

Comment:
Maybe we can move this into `JobsListView` to avoid this custom padding. The top padding could be done with ` EuiSpacer`. Note that the anomaly jobs list has a padding of `20px` which is different from the `16px` the analytics page has.

Reply:
Done in https://github.com/elastic/kibana/pull/49464/commits/7bd5ad5cff8e55d4a717601ccd384796112858d9